hs_core/hydroshare/utils.py

```diff
@@ -125,7 +126,7 @@ def serialize_science_metadata(res):
     resd['files'] = [dc['fields'] for dc in json.loads(js.serialize(res.files.all()))]
     return json.dumps(resd)
 
-
+# TODO: Tastypie left over. This needs to be deleted
```

Comment:
@pkdash Off hand I don't know why some of the tests are running twice on Jenkins.  Perhaps we can hold a developer call today to look at it?  @mjstealey Can you spin up a WebEx for this?